cpp/unreal_plugins/UrdfBot/Source/UrdfBot/UrdfJointComponent.h

```diff
@@ -22,6 +23,13 @@ class UUrdfJointComponent : public UPhysicsConstraintComponent
 public:
     void initializeComponent(UrdfJointDesc* joint_desc, UUrdfLinkComponent* parent_link, UUrdfLinkComponent* child_link);
 
+    void applyAction(float action);
+    void addAction(float action);
+
+    UUrdfLinkComponent* parent_link_;
+    UUrdfLinkComponent* child_link_;
+
 private:
```

Comment:
Maybe make these variables public since we're doing it for parent_link and child_link, and I could imagine a user of this class wanting to be able to query the joint type and the control type.

Reply:
fixed